### ISSUE TITLE
Unexpected multiline string format with `wrap`

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#648]
With this code:

```swift
_ = [
    (foo,
"""
    foo
"""),
    (bar,
"""
    bar
"""),
]
```

If you format with: `swiftformat --rules wrap --maxwidth 110 foo.swift --swiftversion 5.2 --cache clear`

You get:

```swift
_ = [
    (foo,
     """
    foo
"""),
    (bar,
     """
    bar
"""),
]
```

Where the string has now changed.

0.44.13

**Comment by nicklockwood** [comment]
@keith I can see why this would be slightly surprising, but are you sure the value of these strings has actually changed?

**Comment by keith** [comment]
ah I think you're right that they haven't, I'm too used to python's semantics here, sorry about the alarmism

**Comment by nicklockwood** [comment]
n/p thanks for all the detailed reports - I'll get those fixed asap.

**Comment by nicklockwood** [comment]
@keith fixed in 0.44.17
